dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/EuTaxonomyDataForFinancialsController.kt

```diff
@@ -2,23 +2,23 @@ package org.dataland.datalandbackend.controller
 
 import com.fasterxml.jackson.databind.ObjectMapper
 import org.dataland.datalandbackend.interfaces.DataManagerInterface
-import org.dataland.datalandbackend.model.EuTaxonomyData
+import org.dataland.datalandbackend.model.EuTaxonomyDataForFinancials
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 /**
- * Controller for the EU Taxonomy endpoints
+ * Controller for the EU Taxonomy endpoints of financial companies
  * @param myDataManager data manager to be used
  * @param myObjectMapper object mapper used for converting data classes to strings and vice versa
  */
-@RequestMapping("/data/eutaxonomies")
+@RequestMapping("/data/eutaxonomy/financials")
 @RestController
```

Comment:
Discuss route layout

Reply:
Vorschlag:
- Team Discussion nach Daily, in dem du deinen geshelveten Vorschlag zeigst
- Je nach Ausgang der Diskussion => Ticket für die Änderung erstellen

Reply:
Siehe DALA-456